src/core/imap/MCIMAPSession.cpp

```diff
@@ -477,6 +486,16 @@ bool IMAPSession::isVoIPEnabled()
     return mVoIPEnabled;
 }
 
+void IMAPSession::setNetEaseWorkaroundEnabled(bool enabled) 
```

Comment:
Could you remove the methods setNetEaseWorkaroundEnabled() and isNetEaseWorkaroundEnabled() since we have an automatic detection?